BrightspaceUI/htmleditor
# feat: US156396 - GenAi - Added Affordance Component

Added an affordance component which is used in Edit HTML screen.

![image](https://github.com/BrightspaceUI/htmleditor/assets/32684226/14a5ffb4-b239-42d4-95b7-54e6bc08f7b3)

Related PR:
https://github.com/BrightspaceHypermediaComponents/activities/pull/4248

To Test:
Enable and disable GenAi and see if affordance shown. Should be reliant on genAiEndpoint. Therefore, only be displayed with GenAi enabled.
Need to test on legacy activity vs new activity

Related Stories
[US156396](https://rally1.rallydev.com/#/?detail=/userstory/719624512103&fdp=true): Add affordance on content page edit screen (split into one per content exp)
